lib/govuk_message_queue_consumer/test_helpers/mock_message.rb

```diff
@@ -1,3 +1,5 @@
+require "ostruct"
```

Comment:
heya, just checking, does this mean you'll be removing the ostruct requirement from other apps?

Reply:
if so it might be worth linking those PRs in the description for this one

Reply:
Ah no - the app I've been having problems with did not actually require `ostruct` at all. This meant it suddenly broke when another one of it's dependencies stopped requiring it.

Reply:
It shouldn't cause issues for it to be additionally required in any other app though!